config/api.js

```diff
@@ -354,7 +354,9 @@ const routes = (BASE_URL) => {
 
       return url;
     },
-
+    PAPER_CENSOR: ({ paperId, isRemoved }) =>
+      BASE_URL +
+      `paper/${paperId}/${isRemoved ? "censor_paper" : "restore_paper"}/`,
```

Comment:
To follow convention, could we have the endpoint be `/censor` and `/restore` ? The paper prefix implies that a paper is the subject.

Reply:
These endpoints were already being used.